internal/e2e/fine_grained_no_bin/index.js

```diff
@@ -1,3 +1,5 @@
 const path = require("path");
+const Module = require("module");
 
 console.log('hello ' + require.resolve("./test.js", { paths: [ path.join(__dirname, 'test') ] }));
+console.log('hello ' + Module._resolveFilename("test", { paths: [ path.join(__dirname, 'test', 'node_modules') ] }));
```

Comment:
I guess the assertion here is just that it doesn't throw? we don't care what we get back from _resolveFilename in any test?

Reply:
Yeah true, I did not think about it, because I thought we don't need to test nodejs' resolving mechanism but it is probably still worth doing - let me add a proper test for it.

Reply:
ping me when you've added a test?

Reply:
@alexeagle Done.